docs/sftt/sftt-api-spec.md

```diff
@@ -72,7 +72,7 @@ Must be called on an open (not marked reserved, completed or QA) block. Will cre
 This block was reserved successfully.
 
 ##### `400 BAD REQUEST`
-If the block id specified is invalid.
+If there is not a block associated with the given block_id OR there is not a team associated with the given team_id OR the user is not a member of the given team OR the block does not have status 'open'.
```

Comment:
⛏️ Might be easier to read in bullet point format

Reply:
❓📌  Should it be an unauthorized issue if the member is not of the given team?

Reply:
Right now this exception is a 400 error which is a bad request. It looks like all of the authorization exceptions we throw specifically have to do with not having the correct privilege levels as opposed to not having access to a resource based on reservation status/ownership etc.  Maybe we want to change these to 403 errors at some point

Reply:
Edit: I deleted a previous comment on here that you can disregard.